libs/atqr/domain/src/lib/domain-spec/concept-set/Contacts.disl

```diff
@@ -3,7 +3,7 @@ domain ATQR
   //--------------------------------//
   // Contacts
   //--------------------------------//
-  concept Email
```

Comment:
This change is semantically and syntatically incorrect!

Reply:
@yfernandes

Reply:
@yfernandes

Reply:
It's syntatically wrong because Disl syntax is to use expands instead of extends.